backend/curation/api/curation-api.yml

```diff
@@ -14,6 +14,27 @@ info:
     ## Data Model
     <!-- Data model diagram created via Lucid: https://lucid.app/lucidchart/8d085832-34f4-4413-9c1d-44e77d06e326/edit?invitationId=inv_ed354532-7256-4e0a-9ad0-b622905e5198&page=Vfzkzdv~I1fL# -->
     <img src="/static/discover-api-data-model.png" width="50%"/>
+
+    ## Working with versions
+
+    #### Access to prior published version history
+    The CZ CELLxGENE Discover API employs the concept of *Versions* for our two core entities, *Collections* and
+    *Datasets*. This is to permit access to prior published instances (versions) of these entities even as new updates
+    for them are published.
+
+    #### Version identifiers
+    While the *Collection* identifier `collection_id` will always refer to the latest published instance of a Collection
+    (like the "head" in version-control terminology), and must be used with `Collection` endpoints, the
```

Comment:
The *Collection* identifier `collection_id` refers to the most recent published version of a Collection.

Reply:
**(like the "head" in version-control terminology)**

Are our consumers assumed to be familiar with version-control terminology? Why would this need to be introduced?

Reply:
No, but it is a useful analogy.

You want to get rid of it?

Reply:
wrt the beginning of the sentence: I think "While..." makes sense since the focus is on explaining the _version_ identifiers, which happens in the second half of the sentence.

Reply:
> No, but it is a useful analogy.
> You want to get rid of it?

Absolutely. It's not required to explain versions and assumes a shared background in version-control.